src/NHibernate.Test/Async/Tools/hbm2ddl/SchemaValidator/SchemaValidateFixture.cs

```diff
@@ -67,14 +68,12 @@ public class SchemaValidateFixtureAsync
 			await (new SchemaExport(cfgV1).ExecuteAsync(true, true, false));
 
 			var validatorV2 = new Tool.hbm2ddl.SchemaValidator(cfgV2);
-			try
-			{
-				await (validatorV2.ValidateAsync());
-			}
-			catch (HibernateException e)
-			{
-				Assert.That(e.Message, Does.StartWith("Missing column: Name"));
-			}
+
+			Assert.That(
+				() => validatorV2.ValidateAsync(),
+				Throws.TypeOf<SchemaValidationException>()
+				      .And.Message.EqualTo("Schema validation failed: see list of validation errors")
+				      .And.Property("ValidationErrors").Contains("Missing column: Name in nhibernate.dbo.Version"));
```

Comment:
As showcased by failing tests, this column naming is specific to SQL Server and causes the test to fail for other db.